docs/source/models/concept_models/vivarium_alzheimers/concept_model.rst

```diff
@@ -532,10 +539,11 @@ scenario, and input draw.
       aggregated results.
 
       * Everything from 2.0, except use FHS values for ACMR in the total mortality calculation
+      * Verify that ACMR decreases slightly from 2025 to 2050 and then levels off
```

Comment:
Do we actually mean ACMR here, or is this ACMR - CSMR + EMR? Or could that quantity also be called "ACMR" in the context of this sim where the entire population consists of people with AD?

Do you know why the mortality decreases and then levels off? Is this what we expect from the FHS data?

Reply:
I will change to ACMR - CSMR + EMR, which is more precise.  Mortality decreases (as I expected) because people are getting a little healthier over time, I suspect it is mostly because of less tobacco use.